sdk/communication/azure-communication-chat/src/samples/app/build.gradle

```diff
@@ -0,0 +1,70 @@
+plugins {
+    id 'com.android.application'
+}
+
+android {
+    packagingOptions {
+        exclude 'META-INF/DEPENDENCIES'
+        exclude 'META-INF/LICENSE'
+        exclude 'META-INF/LICENSE.txt'
+        exclude 'META-INF/LICENSE.md'
+        exclude 'META-INF/license.txt'
+        exclude 'META-INF/NOTICE'
+        exclude 'META-INF/NOTICE.txt'
+        exclude 'META-INF/NOTICE.md'
+        exclude 'META-INF/notice.txt'
+        exclude 'META-INF/ASL2.0'
+    }
+
+    compileSdkVersion 30
+    buildToolsVersion "30.0.1"
+
+
+    defaultConfig {
+        applicationId "com.azure.chattestapp"
+        minSdkVersion 21
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.0"
+
+        buildConfigField "String", "COMMUNICATION_SERVICE_ENDPOINT", "\"${System.env.COMMUNICATION_SERVICE_ENDPOINT}\""
+        buildConfigField "String", "COMMUNICATION_SERVICE_ACCESS_KEY", "\"${System.env.COMMUNICATION_SERVICE_ACCESS_KEY}\""
+        buildConfigField "String", "COMMUNICATION_CHAT_THREAD_MEMBER_1", "\"${System.env.COMMUNICATION_CHAT_THREAD_MEMBER_1}\""
+        buildConfigField "String", "COMMUNICATION_CHAT_THREAD_MEMBER_2", "\"${System.env.COMMUNICATION_CHAT_THREAD_MEMBER_2}\""
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+}
+
+dependencies {
+
+    api "net.sourceforge.streamsupport:android-retrofuture:1.7.3"
+
+    api "com.azure.android:azure-core:$azureCoreVersion"
```

Comment:
This gradle can only work with a parent gradle where $azureCoreVersion is defined. If we were to publish the sample on its own, probably should only reference the published versions